CODE/logicytics/__init__.py

```
@@ -2,2 +1,29 @@
+from logicytics.Execute import Execute
+from logicytics.Get import Get
+from logicytics.Logger import Log
```

Ruff (0.8.0)

`logicytics.Logger.Log` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)